src/init.cpp

```diff
@@ -978,6 +979,7 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
 
     // Initialize elliptic curve code
     ECC_Start();
+    verify_handle.reset(new ECCVerifyHandle());
```

Comment:
Nice to see this has an explicit lifecycle, instead of the OpenSSL global init object